rtribbia/homework.txt

```diff
@@ -0,0 +1,86 @@
+1. a) "1" == 1
+	true: == is equal-ish
+
+	b) "1" === 1
+		false: "string" vs "number", === is stricter
+
+	c) x == 'x'
+		true: x is set to 'x'
+		false: all other instances
+
+	d) x == (x+'')
+		true: even if x is initially a "number" because same reason as in b)
+
+	e) '' == ' '
+		false: ' ' is not empty
+
+	f) x = true
+		true: this is assignment not equality
+
+	g) var x; x == 'undefined'
+		true
+
+	h) '9'<'10'
+		false: first character, 9 > 1
+
+	i) typeof x + 1 === "number"
+		false: (typeof x) + 1 produces string
+
+	j) typeof x % 2 === "number"
+		false: always- string mod 2 will produce NaN
+
+	k) typeof (x % 2) === "number"
+		true: if x is a number
+		false: if x is not a number
```

Comment:
`string`&2` will actually return `NaN`, which (despite what you might think) is a number--so this is just true.